### DISCUSSION TITLE
Declare proper Pygments styles

### DISCUSSION THREAD

**Discussion by webknjaz** [pradyunsg/furo#156] [Potential Issue] [answered]
Currently, the CSS for code highlighting is hardcoded as a file called `pygments.css`. This is rather suboptimal and is painful to maintain.

Instead, there should be a `pygments.style.Style` subclass with the token mappings. Example: https://github.com/ansible-community/ansible-pygments/blob/main/src/ansible_pygments/styles.py.

As a bonus, this class would be usable for any Pygments integration, even if it does not use HTML/CSS at all (like in the case of terminal output).

**Comment by pradyunsg** [comment, marked as answer]
That comes from Sphinx, and there's nothing that Furo can do to resolve this.

https://github.com/sphinx-doc/sphinx/blob/df71e62dd8c66109e8b5839d8e66f42f8c927c24/sphinx/builders/html/__init__.py#L296

**Reply by webknjaz** [reply]
Weird, I was pretty sure furo had its own style to match the theme.

**Reply by pradyunsg** [reply]
Nope. And even if it did, it'd still use the same approach as Sphinx to not be drastically more work for me. :)
